models/boards.js

```diff
@@ -409,6 +409,21 @@ Boards.helpers({
   },
 
   lists() {
+    const enabled = Meteor.user().hasShowDesktopDragHandles();
```

Comment:
showDesktopDragHandles in separate feature, it should not be used for sorting

Reply:
this is by logical, if user has chosen using draghandle to sort his list, then you don't need to add other sort method.